### ISSUE TITLE
Skip links

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1904]
> Skip links should be visible when they receive keyboard focus (i.e, are tabbed to). Recommend changing "skip to content", "skip to navigation" and "sitemap" links to be visible when they receive focus. See WebAIM's resource on temporarily hidden skip links for help.
